### ISSUE TITLE
Cannot connect to Mailhog

### ISSUE THREAD

**Issue by malcomio** [scotch-io/scotch-box#389]
The box seems to have installed as expected, and http://192.168.33.10 says the Mailhog "is turned on automatically now"

However, http://192.168.33.10:8025/ refuses browser connections

**Comment by malcomio** [comment]
possibly related to #369?
I'd previously installed an older version, and trying to update the box gives the following error:

```
➜  scotch2 git:(master) vagrant box update
==> default: Checking for updates to 'scotch/box'
    default: Latest installed version: 2.5
    default: Version constraints:
    default: Provider: virtualbox
There was an error while downloading the metadata for this box.
The error message is shown below:

The requested URL returned error: 404 Not Found
```

**Comment by malcomio** [comment]
The fix was as suggested in #367 - this is what I did:

1. `vagrant destroy`
2. `vagrant box  remove scotch/box`
3. edit the Vagrantfile and add `Vagrant::DEFAULT_SERVER_URL.replace('https://vagrantcloud.com')` 
4. `vagrant up`
